pkg/sql2pgroll/create_table.go

```diff
@@ -41,7 +41,7 @@ func convertCreateStmt(stmt *pgq.CreateStmt) (migrations.Operations, error) {
 
 	return migrations.Operations{
 		&migrations.OpCreateTable{
-			Name:    stmt.Relation.Relname,
+			Name:    stmt.Relation.GetRelname(),
```

Comment:
Do we need to support qualified names here?

Reply:
Thanks 🙏 I fixed it with a separate PR: https://github.com/xataio/pgroll/pull/557